mockgcp/Makefile

```diff
@@ -26,6 +26,7 @@ gen-proto:
 		--grpc-gateway_opt paths=source_relative \
 		--experimental_allow_proto3_optional \
 		./apis/mockgcp/cloud/resourcemanager/v1/*.proto \
+		./third_party/googleapis/mockgcp/api/apikeys/v2/*.proto \
```

Comment:
Nit: Maybe we can add a lint/a unit test to ensure the paths are ordered alphabetically?

Reply:
We should aim to but I don't think we need to enforce it.  (It's also tricky to enforce)